src/AWS.Logger.Log4net/AWS.Logger.Log4net.csproj

```diff
@@ -35,11 +35,13 @@
   <ItemGroup>
     <ProjectReference Include="..\AWS.Logger.Core\AWS.Logger.Core.csproj" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.5' ">
-      <PackageReference Include="log4net" Version="2.0.8" />
```

Comment:
Why was this removed?

Reply:
the version for log4net was common to both netstandard1.5 and net45. So I have removed the framework conditinal